src/resolvers/Mutation/removeOrganization.ts

```diff
@@ -27,7 +27,7 @@ export const removeOrganization: MutationResolvers["removeOrganization"] =
   async (_parent, args, context) => {
     const currentUser = await User.findOne({
       _id: context.userId,
-    }).lean();
```

Comment:
You talked about using `lean()` only when we're sure we're not going to use any of the other functionalities mongoose provides on the document, but I don't see you using any of those properties. What was it removed then?

Reply:
If we want to modify the data returned by the query, we should not use lean(). The returned object will not have the methods provided by Mongoose to modify and save the document.

Reply:
Sir maybe in the future, we will use save(), virtual, getters,etc. According to my experience, I have not seen any query using lean(), If we want to modify the data returned by the query, we cannot use lean(). The returned object will not have the methods provided by Mongoose to modify and save the document.
But, I can revert this change sir. Waiting for your call on that!?

Reply:
@nishant3721 In the future if it's needed the simple fix is remove the `lean()` method, it's not that hard to do. Nowhere in the code you're using those mongoose document functions, so there's no point of removing the `lean()` method. It's put there in the first place because those things are not being used.

Reply:
Sure I will revert this change in the pr